public/style.css

```diff
@@ -433,11 +435,58 @@ input[type=checkbox]:checked + label ~ .talent-list {
   }
 }
 
+@media only screen and (width: 667px) {
+  html {
+    font-size: 28.95%
+  }
+
+  .about-image-container {
+    position: relative;
+    top: 4.5rem;
+  }
+
+  .projects-title {
+    top: 16rem;
+  }
+
+  .teal-triangle1 {
+    width: 35rem;
+    height: 26rem;
+  }
+
+  .teal-triangle2 {
+    width: 35rem;
+    height: 26rem;
+    left: 39.15rem;
+  }
+
+  .more-projects-button {
+    width: 10rem;
+    height: 4rem;
+    top: -82rem;
+    left: 60rem;
+  }
+
+  .more-learners-button {
+    width: 10rem;
+    height: 4rem;
+    left: -2.5rem;
+  }
+}
+
 @media only screen and (min-device-width: 768px) and (max-device-width: 1024px) {
```

Comment:
I like to follow the convention of `min-device-width` being where the range starts (so `768px` in this case, as written), with`max-device-width` being set to the beginning of the next range *minus one*. 

Thus `max-device-width: 1023px`

This way, a screen that is exactly 1024px wide will use the next step up, which I assume has a `min-device-width: 1024px` media query.

If the screen is 1023.476274872 pixels wide, it'll break =\ I choose (at this moment) to ignore this for now so we can move on in the project.